Tests/AppTests/__Snapshots__/WebpageSnapshotTests/test_home.1.txt

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"/>
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css" integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO" crossorigin="anonymous"/>
+    <link href="/grid.css" rel="stylesheet"/>
+    <title>Swift Package Index</title>
+    <meta name="twitter:title" content="Swift Package Index"/>
+    <meta name="og:title" content="Swift Package Index"/>
+  </head>
+  <body>
+    <div class="container">
+      <div class="row">
+        <h2>Swift Package Index</h2>
+      </div>
+      <form action="/packages/run/reconcile" method="get">
+        <div class="row">
+          <input class="btn btn-primary" type="submit" value="Reconcile"/>
+        </div>
+        <div class="row">
+          <label>Reconcile the master package list with the Package Index.</label>
+        </div>
+      </form>
+      <form action="/packages/run/ingest" method="get">
+        <div class="row">
+          <input class="btn btn-primary" type="submit" value="Ingest"/>
+        </div>
+        <div class="row">
+          <label>Ingest metadata for a batch of packages.</label>
+        </div>
+      </form>
+    </div>
+  </body>
+</html>
```

Comment:
Nice - the html snapshot only shows cosmetics differences now (the baseline commit is 63eee1325bb54d2879e86253a87370597fd59fe2) and the image is identical against that revision rendered with the previous dsl. All good now!